pkg/testing/cmd_test.go

```diff
@@ -0,0 +1,142 @@
+package testing
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"gopkg.in/check.v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/kanisterio/kanister/pkg/kopia/command"
+	"github.com/kanisterio/kanister/pkg/kopia/repository"
+	"github.com/kanisterio/kanister/pkg/kube"
+	"github.com/kanisterio/kanister/pkg/testutil"
+)
+
+const (
+	testPodName = "test-kopia-cmd-"
+)
+
+type KopiaCmdSuite struct {
+	cli       kubernetes.Interface
+	namespace string
+}
+
+var _ = check.Suite(&KopiaCmdSuite{})
+
+func (s *KopiaCmdSuite) SetUpSuite(c *check.C) {
+	config, err := kube.LoadConfig()
+	c.Assert(err, check.IsNil)
+	cli, err := kubernetes.NewForConfig(config)
+	c.Assert(err, check.IsNil)
+	s.cli = cli
+
+	ctx := context.Background()
+	ns := testutil.NewTestNamespace()
+	ns.GenerateName = "kanister-datatest-"
+
+	cns, err := s.cli.CoreV1().Namespaces().Create(ctx, ns, metav1.CreateOptions{})
+	c.Assert(err, check.IsNil)
+	s.namespace = cns.GetName()
+
+}
+
+func (s *KopiaCmdSuite) TearDownSuite(c *check.C) {
+	ctx := context.Background()
```

Comment:
[nit] Better to use `context.TODO()` in cleanup actions as we always want to execute that regardless of the parent context event.